Gettext_helpers.cmake

```
@@ -97,8 +97,8 @@ function(configure_gettext)
                 "--output=${GETTEXT_POTFILE_DESTINATION}/${GETTEXT_DOMAIN}.pot"
             WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
     endif()
-    add_custom_command(
-        OUTPUT "${GETTEXT_POTFILE_DESTINATION}/${GETTEXT_DOMAIN}.pot"
+    # should use add_custom_target so .pot doesn't get cleaned
+    add_custom_target(${TARGET_NAME}_gen_pot
```

```suggestion
    add_custom_target(${TARGET_NAME}_gen_pot
        BYPRODUCTS "${GETTEXT_POTFILE_DESTINATION}/${GETTEXT_DOMAIN}.pot"
```
for parity with `add_custom_command`'s `OUTPUT`; same applies to line 129 (per-language `.po` file targets)